internal/controller/components/kserve/kserve.go

```diff
@@ -47,11 +47,12 @@ var (
 )
 
```

Comment:
think in https://github.com/robotmaxtron/opendatahub-operator/blob/5f912d7b3974d31d10faab328415730cfde35000/internal/controller/components/kserve/kserve_controller.go#L73 we need to add a watch on CRD `LeaderWorkerSetOperator`